Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst

```diff
@@ -79,8 +79,8 @@ Adding a File
 This example adds a new file in the root folder of the default
 Storage::
 
-   $storageRepository = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Core\Resource\StorageRepository::class);
-   $storage = $storageRepository->getDefaultStorage();
+   $resourceFactory = \TYPO3\CMS\Core\Resource\ResourceFactory::getInstance();
```

Comment:
```suggestion
   $resourceFactory = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Core\Resource\ResourceFactory::class);
```

See https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.3/Deprecation-90260-ResourceFactorygetInstancePseudo-factory.html

Reply:
Thank you. I thought about this and deciced to stay consistent with the documentation of `9.5`, as the documentation changes with `>10.4` anyway.

But yeah, makes sense to don't recommend deprecated stuff.